src/getMetroConfig.js

```diff
@@ -8,49 +8,49 @@
 const fs = require('fs')
 const exec = require('child_process').execSync
 const dedent = require('dedent-js')
-const getDependencyPath = require('./getDependencyPath')
 
 const mapModule = name =>
     `'${name}': path.resolve(__dirname, 'node_modules/${name}')`
 
-const mapPath = path =>
-    `/${path.replace(
-        /\//g,
-        '[/\\\\]',
-    )}[/\\\\]node_modules[/\\\\]react-native[/\\\\].*/`
+const mapPath = path => `/${path.replace(/\//g, '[/\\\\]')}[/\\\\].*/`
```

Comment:
💁 We only want to add react-native to the blacklist where it's actually a peer dependency. Maybe we should rename this method, too. Too many "mapPath"s around this codebase.